common/src/IO/MipTextureReader.cpp

```diff
@@ -67,7 +67,7 @@ namespace TrenchBroom {
                 Assets::Palette palette = doGetPalette(reader, offset, width, height);
 
                 if (!palette.initialized()) {
-                    return nullptr;
+                    return new Assets::Texture(textureName(name, path), width, height);
```

Comment:
What kind of failure does this correspond to, truncated palette data?

Reply:
The crash we observed here was because the skin loader didn't handle the null values. The usual texture loaders are fine with return nullptr, but I think you are right, we should always return a value. I'll revise this tonight.

Reply:
I have removed all the cases in which nullptr is returned, and now we return an empty placeholder texture instead.